src/new/wallet.js

```diff
@@ -1454,11 +1475,13 @@ class HathorWallet extends EventEmitter {
    *    'skipSpent': if should not include spent utxos (default true)
    *  }
    *
-   * @return {Array} Array of objects with {tx_id, index, address} of the authority output. Returns null in case there are no utxos for this type
+   * @return {Array} Array of objects with {txId, index, address} of the authority output. Returns an empty array in case there are no tx_outupts for this type
```

Comment:
Returns a promise now that the method is async

Reply:
Done in 5b788957fd1124ee7f4f2f51eda098737313bb7a